src/test/resources/com/puppycrawl/tools/checkstyle/checks/blocks/needbraces/InputNeedBracesEmptyDefault.java

```diff
@@ -17,3 +17,7 @@ private void main1() {
 @interface Example {
     String priority() default "value";
 }
+
+interface IntefaceWithDefaultMethod {
+    default void doIt(){}
```

Comment:
confirmed, this Input file is used by both UT, on default config and on CASE and DEFAULT - https://github.com/checkstyle/checkstyle/blob/a73ff0890c27cbe8affc1a55f2710a231687bd85/src/test/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheckTest.java#L148

Reply:
Sorry, don't understand what you mean and what corrections required.

Reply:
No changes are required, I left note for myself